### PR TITLE
torch.special.gamma

### PR DESCRIPTION
**Pull request by 0x00b1**
```Python
gamma(input, *, out=None) -> Tensor
```

Gamma function $\Gamma\left(\text{input}\right)$.

### CODE REVIEW

File: torch/special/__init__.py [hunk 1]
```diff
@@ -1014,6 +1015,20 @@
     {out}
 """.format(**common_args))
 
+gamma = _add_docstr(_special.special_gamma,
+                    r"""
+gamma(input, *, out=None) -> Tensor
+
+Gamma function :math:`\Gamma\left(\text{input}\right)`.
```
Comment (mruberry) [inline torch/special/__init__.py:1022]: I know we'll come back for the docs one day, but elaborating here would be nice

Comment (0x00b1) [reply torch/special/__init__.py]: Updated!

File: torch/testing/_internal/common_methods_invocations.py [hunk 2]
```diff
@@ -19193,6 +19193,34 @@ def error_inputs_mean(op_info, device, **kwargs):
         supports_one_python_scalar=True,
         supports_autograd=False,
     ),
+    UnaryUfuncInfo(
+        'special.gamma',
+        decorators=(
+            precisionOverride(
```
[the remark this answers is not in the document]
Comment (0x00b1) [reply torch/testing/_internal/common_methods_invocations.py]: Fixed!

File: torch/testing/_internal/common_methods_invocations.py [hunk 3]
```diff
@@ -19193,6 +19193,34 @@ def error_inputs_mean(op_info, device, **kwargs):
...
+                {
+                    torch.float32: 1e-03,
+                    torch.float64: 1e-05,
+                },
+            ),
+        ),
+        dtypes=all_types_and(torch.bool),
+        ref=scipy.special.gamma if TEST_SCIPY else _NOTHING,
+        skips=(
+            DecorateInfo(
```
[the remark this answers is not in the document]
Comment (0x00b1) [reply torch/testing/_internal/common_methods_invocations.py:19715]: Oops. Missed this comment. Would you mind if I added a comment in a follow-up? If you are curious, like many gamma implementations, things get wonky outside of the [-5, 5] interval.

**Review by mruberry — APPROVED** [review]
Cool! Just needs a rebase

**Comment by 0x00b1** [comment]
@suo Are you sure? I don't see how the `TestFakeQuantizeOps` failure is related.

**Comment by suo** [comment]
I don't either, but after I reverted this PR the test started passing again

**Pull request closed without merging**